app/views/admin/dashboard/dashboard.html.slim

```diff
@@ -0,0 +1,14 @@
+section.dashboard
+  h1 = @person.name
+
+  section.block.upcoming-events.calendar
+    h2 Upcoming Steps
+    div#calendar
+
+javascript:
+  $(function(){
+    MS.DashboardCalendarView = new MS.Views.DashboardCalendarView({
```

Comment:
I don't think you need to assign this to anything here. We do with the other CalendarView for separate reasons.